pandasai/helpers/code_manager.py

```diff
@@ -46,12 +46,12 @@ def __init__(
         if self._config.middlewares is not None:
             self.add_middlewares(*self._config.middlewares)
 
-    def add_middlewares(self, *middlewares: List[Middleware]):
+    def add_middlewares(self, *middlewares: Optional[Middleware]):
         """
         Add middlewares to PandasAI instance.
 
         Args:
-            *middlewares: A list of middlewares
+            *middlewares: Middlewares to be added
 
         """
         self._middlewares.extend(middlewares)
```

Comment:
@nautics889

Reply:
@gventuri 
Actually, i don't think it really matters. Let me show your an example:
```python
>>> middlewares_list = []
>>> def add_middleware(*middlewares):
...     middlewares_list.extend(middlewares)
>>>     
>>> add_middleware()  # this works, no exception was raised
>>> add_middleware(1)  # this works as well
>>> add_middleware(2, 3, 4)  # also works 
>>> print(middlewares_list)  # output
[1, 2, 3, 4]
```
_(the code above essentially imitates our case)_

This happens because when you declare a function/method with single asterisk notation, the positional argumets will always be within `tuple` structure, no matter how many arguments you pass when calling: zero or 999, they supposed to be "wrapped in tuple". So, as i understand, we can leave `.extend()` there without any additional condition checks, since there can only be a tuple to be passed.

I understand, @coderabbitai confused me as well, but in this there is no need to add those conditions.

Let me know if i'm mistaken pls.

Reply:
@nautics889 yeah, you're right, 100% agree! Let's keep it like that then! Thanks a lot, merging 🚀